### DISCUSSION TITLE
五笔输入方案的调频问题

### DISCUSSION THREAD

**Discussion by X-hood** [rime/home#810] [Q&A]
官网下载的五笔方案，默认重码词条顺序不是太理想，于是启用了调频。但有时候，重码的两个词都经常使用，结果就是它们的顺序来回变动，有时候打顺手了就常常打错字。
有没有设置能在调整过频率一段时间后，停止继续调频，但保存当前的词条频率，而不是回到默认的顺序。

百度类似问题，有说给词库中词条加一个较大的权重的，感觉不是很方便，打开文件根本不知道要修改哪些词条的权重，总不能打到一个编码想要调整的时候就去码表中修改权重值吧。

有没有可能设置停用调频，但在重码时，在输入界面选中一个词条，手动调整它的位置次序（前移、后移、置顶之类的）？

**Comment by xtay573269555** [comment]
确实挺需要这个功能的。如果能加入快速删除词条功能就更好了，因为有些自动生成的词条是不需要或者不对的
